chakracore/10/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM buildpack-deps:stretch
 RUN groupadd --gid 1000 node \
   && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
 
-ENV NODE_VERSION 10.13.0
+ENV NODE_VERSION 10.0
```

Comment:
Is this desired?

Reply:
no - but it didn't happen when I re-ran the script. Weird...